packages/@okta/vuepress-site/docs/guides/sign-users-out/sign-out-of-okta/android/remotesignout.md

```diff
@@ -2,25 +2,65 @@
 
 Signing out of Okta requires the app to open a browser and navigate to the [end session endpoint](https://developer.okta.com/docs/reference/api/oidc/#logout). Okta destroys the user's session and immediately redirects back to your application.
 
-To do this, you must define a callback route for the sign-out process. 
-
-`TODO: Instructions to add signout redirect URI to gradle file`
-
-Next, add it as an allowed **Logout redirect URI** in the Okta Developer Console:
+To do this, you must define a callback route for the sign-out process.
+This should of been done in the previous step: [Create an Okta application](https://developer.okta.com/docs/guides/sign-into-mobile-app/android/create-okta-application/)
+but if you haven't done it:
```

Comment:
And then add this sentence to introduce the steps to adding that callback route as an allowed logout redirect URI:

Next, add it as an allowed **Logout redirect URI** in the Okta Developer Console: